assets/src/components/ProgressBarV2.js

```diff
@@ -6,18 +6,16 @@ import { roundToXDecimals } from '../util/math'
 
 const styles = theme => ({
   gradedBar: {
-    float: 'left',
     backgroundColor: theme.palette.secondary.main
   },
 
   ungradedBar: {
-    float: 'left',
     backgroundColor: theme.palette.info.main
   },
 
   outOfBar: {
-    float: 'left',
-    backgroundColor: theme.palette.negative.main
+    // for Jest test the negative palette attributes is not available so adding the condition
+    backgroundColor: theme.palette.negative ? theme.palette.negative.main : '#B5B5B5'
```

Comment:
I'm still reviewing the PR, but I became curious about how to deal with this better. Here's a proposed solution: https://github.com/pushyamig/my-learning-analytics/pull/5